tests/microbenchmarks/bench_moe_comm.py

```
@@ -825,6 +825,79 @@ def _gather_per_rank(times_us: List[float], iter_stats: bool = False) -> Dict[st
+    # warmup->timing call pattern (NCCL_EP especially relies on this).
+    shape = list(recv_hs.shape)
+    shape[-1] = hidden_size
+    moe_out = torch.zeros(tuple(shape), dtype=torch.bfloat16, device=recv_hs.device)
+    backend.combine(moe_out, all_rank_max_num_tokens=max(all_rank_num_tokens))
+    torch.cuda.synchronize()
+
+    first_col = recv_hs[:, 0].to(torch.float32)
+    decoded = first_col.round().to(torch.int64)
+    unique, counts = decoded.unique(return_counts=True)
+    histogram: Dict[int, int] = {
+        int(u) - 1: int(c) for u, c in zip(unique.tolist(), counts.tolist())
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)